contrib/translations/ES.json

```diff
@@ -0,0 +1,28 @@
+{
+	"language": "ES",
+	"list": [
+	{
+		"1": "No se ha encontrado ningún archivo de configuración del usuario o del sistema, utilizando los valores predeterminados de la aplicación",
+		"2": "Archivo de configuración del sistema cargado con éxito",
+		"3": "El archivo de configuración del sistema tiene errores - por favor, compruebe su configuración",
+		"4": "Archivo de configuración cargado con éxito",
+		"5": "El archivo de configuración tiene errores - por favor, compruebe su configuración",
+		"6": "Usando la opción de configuración para los puntos finales globales de Azure AD",
+		"7": "Usando la opción de configuración de Azure AD para los puntos finales del Gobierno de los Estados Unidos",
+		"8": "Usando la opción de configuración de Azure AD para los puntos finales del Gobierno de los Estados Unidos (DOD)",
+		"9": "Usando la opción de configuración para Azure AD Alemania",
+		"10": "Usando la opción de configuración para Azure AD China operado por 21Vianet",
+		"11": "Punto final de Azure AD desconocido: uso de puntos finales de Azure AD globales",
```

Comment:
I haven't seen endpoint translated as "punto final" in any book. I recall "extremo de red" or something like that. Was this checked? maybe looking at how onedrive/windows call it (haven't used a spanish os in a while)

Reply:
Thanks for the review, however this PR depends on another PR which is kinda dead at the moment due to the rebuild and work going on to support v2.5.x